### ISSUE TITLE
[BITV] Share tab: "Copy to clipboard" should be button instead of link

### ISSUE THREAD

**Issue by Pytal** [nextcloud/server#35505]
- [x] "Copy to clipboard" should be button instead of link https://github.com/nextcloud/server/pull/35240
- https://github.com/nextcloud/server/pull/35527
- [x] https://github.com/nextcloud/server/pull/35533

**Comment by Pytal** [comment]
@michaelnissenbaum within the share tab we have a "Copy to clipboard" button shown below
![image](https://user-images.githubusercontent.com/24800714/204675599-c53002a7-777b-47b8-af5e-eb949fb7e9a9.png)

But due to the functionality requirements listed below we have the element set to the `a` tag rather than the `button` tag

- If the copy fails for browser security reason, you cannot get the link.
- The fact that it was a link also allowed you to right/middle click and copy or open in a new tab.

Is using the `button` tag for this absolutely necessary to pass the accessibility audit?

**Comment by michaelnissenbaum** [comment]
As copy to clipboard is an action it's expected that interactive element is implemented using role button. But in your case we'd accept that link element is used.
